Source/cmGlobalFastbuildGenerator.cxx

```diff
@@ -2185,6 +2184,24 @@ class cmGlobalFastbuildGenerator::Detail::Generation
 					Wrap(dependencies, "'", "-" + configName + "'"));
 			}
 
+			context.fc.WritePopScope();
+		}
+
+		// Output the prebuild/Prelink commands
+		WriteCustomBuildSteps(context, lg, target, target.GetPreBuildCommands(), "PreBuild", dependencies);
+		WriteCustomBuildSteps(context, lg, target, target.GetPreLinkCommands(), "PreLink", dependencies);
+
+		// Iterate over each configuration
+		// This time to define prebuild and post build targets for each config
+		configIter = context.self->GetConfigurations().begin();
+		configEnd = context.self->GetConfigurations().end();
+		for (; configIter != configEnd; ++configIter)
+		{
+			const std::string & configName = *configIter;
+
+			context.fc.WriteVariable("BaseCompilationConfig_" + configName, "");
+			context.fc.WritePushScopeStruct();
+
```

Comment:
here, there should be this:
context.fc.WriteCommand("Using", ".BaseConfig_" + configName);